clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -491,6 +521,19 @@ def write_adni_sessions_tsv(df_subj_sessions, bids_subjs_paths):
 
     df_subj_sessions = df_subj_sessions.fillna("n/a")
 
+    # compute the amyloid and ptau status
+    df_subj_sessions[
+        ["adni_av45", "adni_pib", "adni_abeta", "adni_ptau"]
+    ] = df_subj_sessions[["adni_av45", "adni_pib", "adni_abeta", "adni_ptau"]].apply(
+        pd.to_numeric, errors="coerce"
```

Comment:
Looking at the [doc](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.to_numeric.html), I see that `coerce` will force non-numeric data to NaN.

However, in the [BIDS specs](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#tabular-files), missing values are encoded as "n/a". 

As a result, I think we should transform to numeric "on the fly" when calling the `compute_amyloid_status` and `computer_ptau_status` instead.

Reply:
Thanks, I have corrected this issue, the data frames columns used as arguments now remain with n/a instead of being changed to NaN

Reply:
I think it is a bit premature to translate n/a values here. Encoding missing values should happen at write time and `DataFrame.to_csv` has a specific argument (`na_rep`) to enable this.

Reply:
@ghisvail  are you referring to l.522? I agree it could be migrated to write time. I created an [issue](https://github.com/aramis-lab/clinica/issues/326) to clean up the code a bit. We could solve it in a new PR.